phpunit.xml.dist

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit colors="true" bootstrap="vendor/autoload.php">
```

Comment:
Revert. You can specify it for now in `.travis.yml` and command line option to PHPUnit. No need to alter default phpunit config file.